culture.md

```diff
@@ -54,9 +54,9 @@ The rule in the Code of Conduct about avoiding discussions about identity labels
 
 There have been recurring discussions about some labels over the lifetime of the slack, particularly when it came to using "bi(sexual/romantic)" vs "pan(sexual/romantic)" as an identity label. Both of these labels may mean different things, and the discussion in the LGBTQ community involving them can be particularly contentious. Our experience with these discussions was that it was very easy to be accusative or dismissive of other (for example, implying or stating that bi-identifying individuals or their label are inherently against trans or non-binary people). These discussions were often more disruptive and stressful than fruitful, and it was clear that sometimes the larger context was intentionally or accidentally ignored (for example, the erasure that both bi and pan people so often face in the queer community).
 
-It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban that, and any such discussion -- this now includes any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct.
+It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban any discussion about someone else's identity. This also includes discussion of other identities as they relate to your own ("I am pan because I am not bi, which means...") -- this policy extends to other identities, such as any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct. All discussions on gender and sexuality contain the expectation that the participants will stay in their own "lane" as it were, and speak only on their own experiences.
 
```

Comment:
I updated the above to include the example. Please note these suggestions reflect my best understanding after reading the original document, as well as observing discussions in `#meta`